sefaria/views.py

```diff
@@ -59,7 +60,7 @@
 from sefaria.google_storage_manager import GoogleStorageManager
 
 
-from reader.views import render_template
+from reader.views import base_props, render_template, text_panels
```

Comment:
The fact that you need to import these here kinda says to me that they are in the wrong place in the first place.... (not sure what to do about that)